rosdep/base.yaml

```diff
@@ -1660,7 +1667,14 @@ libpcl-all-dev:
       packages: [pcl]
   macports: [libpcl]
   opensuse: [pcl]
-  ubuntu: [libpcl-1.7-all-dev]
+  ubuntu:
+    precise: [libpcl-1.7-all-dev]
+    quantal: [libpcl-1.7-all-dev]
+    raring: [libpcl-1.7-all-dev]
+    saucy: [libpcl-1.7-all-dev]
+    trusty: [libpcl-1.7-all-dev]
+    utopic: [libpcl-1.7-all-dev]
+    vivid: [libpcl-dev]
```

Comment:
It's pretty confusing that the portable coroutine library is `libpcl1-dev` whereas PCL is `libpcl-dev`, surely that isn't intentional...